README.md

```diff
@@ -117,4 +117,43 @@ new HtmlWebPackPlugin({
   title: ...
 ```
 
-A good idea would be to conditionally change `template` path based on external config or an env variable.
\ No newline at end of file
+A good idea would be to conditionally change `template` path based on external config or an env variable.
+
+## Config
+
+### TCR (Optional)
+
+Tokens are dynamically loaded from the contract, but it might not be desirable to display everything in the interface.
+
+To dynamically control which tokens are displayed without the need of a redeployment, it's possible to use a Token Curated Registry (TCR) contract per network.
+
+The only requirement is that this contract implements the following method:
+
+```sol
+function getTokens(uint256 _listId) public view returns (address[] memory)
+```
+
+For a sample implementation, refer to [dxDAO's TCR](https://github.com/nicoelzer/dxDAO-Token-Registry/blob/master/contracts/dxTokenRegistry.sol).
+
+<!-- TODO: use a central place for all configs https://github.com/gnosis/dex-react/issues/978 -->
```

Comment:
https://github.com/gnosis/dex-react/pull/983#discussion_r419327002

> It's ok to add a link to the ongoing task, but better make it in MARKDOWN instead of using HTML comments

Reply:
This was just not meant to be part of the exposed parsed page. I used the editor's comment shortcut.
Besides, next thing I'm currently working on addresses that TODO.